buildSrc/src/main/kotlin/org/jetbrains/exposed/gradle/Versions.kt

```diff
@@ -21,6 +21,9 @@ object Versions {
     const val sqlserver = "9.4.1.jre8"
 
     /** Spring **/
-    const val springFramework = "5.3.29"
-    const val springBoot = "2.7.14"
+    const val springFramework = "6.0.11"
+    const val springBoot = "3.1.3"
+
+    /** Test Dependencies **/
+    const val testContainers = "1.17.3"
```

Comment:
@rbraeunlich This probably got reintroduced when conflicts were being resolved. We don't need `testContainers` anymore, so please remove these lines.

Reply:
@rbraeunlich Also please remove the unused imports in `ExposedAutoConfiguration` that are failing the Detekt action (see details below).

Reply:
All done